src/app/_components/Footer.tsx

```diff
@@ -72,7 +72,7 @@ export const Footer: FC = () => {
             >
               Submit report or request
```

Comment:
```suggestion
              Submit bug or feature request
```

While we are at this, let's ensure this language clarifies why/when someone would open a new issue.

Reply:
done